server/graphql/v2/object/Event.js

```diff
@@ -51,6 +52,16 @@ export const Event = new GraphQLObjectType({
           }
         },
       },
+      endsAt: {
```

Comment:
Why not introduce `startsAt`? `endsAt` without `startsAt` looks a bit like a missed opportunity.

Reply:
Well, my thinking was that we are not using `startsAt` at this stage. Only the `endsAt` is useful for this PR (to support the warning for past events). I normally go by the rule that I try not to introduce changes which are not relevant to the issue in question which the PR aims to solve. This is why I didn't introduce it. Anyways, do you think we should introduce `startsAt`? 🤔

Reply:
It doesn't make sense to introduce `endsAt` alone. I'm asking you to introduce `startsAt` too.

Reply:
@znarf : I've added `startsAt` as well. 👍🏽